go.mod

```diff
@@ -11,15 +11,15 @@ require (
 	github.com/onsi/gomega v1.33.1
 	github.com/openshift/oadp-operator v1.0.2-0.20250227152435-9267da21ab64
 	github.com/sirupsen/logrus v1.9.3
-	github.com/stretchr/testify v1.9.0
+	github.com/stretchr/testify v1.10.0
 	github.com/vmware-tanzu/velero v1.14.0
 	go.uber.org/zap v1.27.0
 	golang.org/x/sync v0.11.0
-	k8s.io/api v0.30.5
-	k8s.io/apimachinery v0.30.5
-	k8s.io/client-go v0.30.5
-	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
-	sigs.k8s.io/controller-runtime v0.18.5
+	k8s.io/api v0.31.3
+	k8s.io/apimachinery v0.31.3
+	k8s.io/client-go v0.31.3
+	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
+	sigs.k8s.io/controller-runtime v0.19.3
```

Comment:
Is this the change that is making tests fail due to name of controller?

wont this break OADP usage itself? Can user install more than one OADP in a cluster with this change?

Reply:
This matches velero dep.

No, conrtroller name registration is contained in controller-manager instance which is contained within each binary run on each pod.

Each OADP pod will have not share this memory of controller names so there is no conflict concerns IMO.